doc/source/building/build_docker_container.rst

```diff
@@ -1,190 +1,91 @@
+ones. KIWI Docker images are considered to be native since the KIWI
+tarball image is ready to be loaded to a Docker daemon, including common
+container configurations.
 
-Building Docker images with KIWI requires a couple of additional tools.
-In practice, :command:`umoci` and :command:`skopeo` utilities are mandatory.
-These tools are not yet in all regular openSUSE repositories, however a packaged
-version can be found in the *Virtualization:containers* repository
-`here <http://download.opensuse.org/repositories/Virtualization:/containers/>`_.
-In order to install them first add the repository using the :command:`zypper ar`
-command:
-
-.. code:: bash
-
-   $ zypper ar -f http://download.opensuse.org/repositories/Virtualization:/containers/<DIST>
-
-where the placeholder `<DIST>` is the preferred distribution. Once the
-repository has been added install the required utilities:
-
-.. code:: bash
-
-   $ zypper in umoci skopeo
-
-Building the Image
-~~~~~~~~~~~~~~~~~~
-
-Run the following KIWI command in order to build the base image example.
-
-.. code:: bash
-
-    $ sudo kiwi-ng --type docker system build \
-        --description kiwi-descriptions/suse/x86_64/suse-leap-42.2-docker \
-        --target-dir /tmp/myimage
-
-Find the image with the suffix :file:`.docker.tar.xz` below
-:file:`/tmp/myimage`. For testing the docker image it's required
-to have the docker subsystem installed and running. The following
-command can be used to check if docker is installed and functional
-on your system:
-
-.. code:: bash
-
-    $ docker images
-
-The command above lists all available docker images. If the docker
-daemon is running and responsive a list output is expected which
-could also be empty if no images are available. In any other case
-an error message is displayed which needs to be resolved prior to
-continue with the next steps.
-
-Once docker works on your system the KIWI built image needs to be
-loaded into the docker system with the following command:
-
-.. code:: bash
-
-    $ docker load -i Docker-Leap-42.2.x86_64-1.0.0.docker.tar.xz
-
-On success the docker image is available as part of the images list as follows:
-
-.. code:: bash
-
-    $ docker images
-
-    REPOSITORY          TAG  ...
-    opensuse            42.2 ...
-
-In order to run an interactive shell session of the loaded container
-the following command needs to be called:
-
-.. code:: bash
-
-    $ docker run -it opensuse:42.2 /bin/bash
-
-Container Configuration
-~~~~~~~~~~~~~~~~~~~~~~~
-
-A docker container includes a set of information also called metadata.
-The metadata information is provided within the KIWI XML description
-as part of the ``<containerconfig>`` section. The following parameters
-can be specified:
+The Docker configuration metadata is provided to KIWI as part of the
+:ref:`XML description file <decription_components>` using the
+``<containerconfig>`` section. The following parameters
```

Comment:
As we talk about XML, we should better use the term "tag" or "element" instead of "section".